target/linux/ramips/image/mt7621.mk

```diff
@@ -468,7 +468,7 @@ define Device/phicomm_k2p
   MTK_SOC := mt7621
   IMAGE_SIZE := 15744k
   DEVICE_VENDOR := Phicomm
-  DEVICE_MODEL := K2P
+  DEVICE_MODEL := K2P/KE2P
```

Comment:
Hi! You could update to the recently added ALT0_MODEL here now :)

Reply:
@rogerpueyo Just put:
DEVICE_VENDOR := Phicomm
DEVICE_MODEL := K2P
DEVICE_ALT0_VENDOR := Phicomm
DEVICE_ALT0_MODEL := KE2P

Reply:
I was pushing to the wrong branch. Here it is :)